var/spack/repos/builtin/packages/veloc/package.py

```diff
@@ -11,43 +11,39 @@ class Veloc(CMakePackage):
     checkpoint-restart runtime for HPC supercomputing infrastructures"""
 
     homepage = "https://github.com/ECP-VeloC/VELOC"
-    url      = "https://github.com/ECP-VeloC/VELOC/archive/1.5.tar.gz"
+    url      = "https://github.com/ECP-VeloC/VELOC/archive/1.6.tar.gz"
     git      = "https://github.com/ecp-veloc/veloc.git"
-
+    maintainers = ['bnicolae']
     tags = ['e4s']
 
-    version('master', branch='master')
+    version('main', branch='main')
+    version('1.6', sha256='451b46ad13e360270044c0dba09d8e4fbd64149f8e8d71310fdb520424c5eeaa')
     version('1.5', sha256='892f3623c73254d40fbbb8cbc3056219a31510e37aae2ede4100c04743701a5c')
     version('1.4', sha256='d5d12aedb9e97f079c4428aaa486bfa4e31fe1db547e103c52e76c8ec906d0a8')
-    version('1.3', sha256='3817ea57045443c1a9a819560911db1175dbe4153e317adaa1492437f3f13f3b')
-    version('1.2', sha256='126a7e01d79458807a6545a8e5f92f8d62a23187dee70be0913b60a1393780e0')
-    version('1.1', sha256='2bbdacf3e0ce4e7c9e360874d8d85b405525bdc7bd992bdb1f1ba49218072160')
-    version('1.0', sha256='d594b73d6549a61fce8e67b8984a17cebc3e766fc520ed1636ae3683cdde77cb')
+    version('1.3', sha256='3817ea57045443c1a9a819560911db1175dbe4153e317adaa1492437f3f13f3b', deprecated=True)
+    version('1.2', sha256='126a7e01d79458807a6545a8e5f92f8d62a23187dee70be0913b60a1393780e0', deprecated=True)
+    version('1.1', sha256='2bbdacf3e0ce4e7c9e360874d8d85b405525bdc7bd992bdb1f1ba49218072160', deprecated=True)
+    version('1.0', sha256='d594b73d6549a61fce8e67b8984a17cebc3e766fc520ed1636ae3683cdde77cb', deprecated=True)
 
-    depends_on('boost')
```

Comment:
Old versions depend on boost. Try `depends_on('boost', when='@:1.5')`, or equivalent if I'm wrong about the version it changed in.

Reply:
Now every version depends on boost